### PR TITLE
Fix error message formatting in paramiko SSH

### PR DESCRIPTION
**Pull request by lakshmi-kannan**

### CODE REVIEW

**Comment by lakshmi-kannan** [comment]
cc: @kirilstrax

**Comment by Kami** [comment]
LGTM, +1

File: st2actions/st2actions/runners/ssh/parallel_ssh.py [hunk 1]
```diff
@@ -242,7 +242,7 @@ def _run_command(self, host, cmd, results, timeout=None):
             results[host] = {'stdout': stdout, 'stderr': stderr, 'return_code': exit_code,
                              'succeeded': is_succeeded, 'failed': not is_succeeded}
         except:
-            error = 'Failed executing command %s on host %s', cmd, host
+            error = 'Failed executing command %s on host %s' % (cmd, host)
```
Comment (Kami) [inline st2actions/st2actions/runners/ssh/parallel_ssh.py]: Also, I assume that this resulted in an exception which was caught in the runner container, right?

Comment (lakshmi-kannan) [inline st2actions/st2actions/runners/ssh/parallel_ssh.py]: It's for an individual host. You'll see error and traceback in the host's result JSON.

**Pull request merged**